worlds/oot_soh/location_access/overworld/lost_woods.py

```diff
@@ -7,6 +7,196 @@
+        (Locations.LW_DEKU_SCRUB_NEAR_DEKU_THEATER_RIGHT, lambda bundle: is_child(bundle) and can_stun_deku(bundle)),
+        (Locations.LW_DEKU_SCRUB_NEAR_DEKU_THEATER_LEFT, lambda bundle: is_child(bundle) and can_stun_deku(bundle)),
+        (Locations.LW_GS_ABOVE_THEATER, lambda bundle: is_adult(bundle) and (
+                    (can_plant_bean(bundle) and can_attack(bundle)) or (
+                        can_do_trick(Tricks.LW_GS_BEAN, bundle) and can_use(Items.LONGSHOT, bundle) or can_use(
+                    Items.FAIRY_BOW, bundle) or can_use(Items.FAIRY_SLINGSHOT, bundle) or can_use(Items.BOMBCHUS_5,
+                                                                                                  bundle) or can_use(
+                    Items.DINS_FIRE, bundle))) and can_get_nighttime_gs(bundle)),
+        (Locations.LW_GS_BEAN_PATCH_NEAR_THEATER,
+         lambda bundle: can_spawn_soil_skull(bundle) and can_attack(bundle) or (
+                     (not world.options.shuffle_scrubs) and can_reflect_nuts(bundle))),
+        (Locations.LW_BOULDER_RUPEE, lambda bundle: blast_or_smash(bundle)),
+        (Locations.LW_BEAN_SPROUT_NEAR_THEATRE_FAIRY1,
+         lambda bundle: is_child(bundle) and has_item(Items.MAGIC_BEAN, bundle) and can_use(Items.SONG_OF_STORMS,
+                                                                                            bundle)),
+        (Locations.LW_BEAN_SPROUT_NEAR_THEATRE_FAIRY2,
+         lambda bundle: is_child(bundle) and has_item(Items.MAGIC_BEAN, bundle) and can_use(Items.SONG_OF_STORMS,
+                                                                                            bundle)),
+        (Locations.LW_BEAN_SPROUT_NEAR_THEATRE_FAIRY3,
+         lambda bundle: is_child(bundle) and has_item(Items.MAGIC_BEAN, bundle) and can_use(Items.SONG_OF_STORMS,
+                                                                                            bundle)),
+        (Locations.LW_AFTER_MIDO_GRASS1, lambda bundle: can_cut_shrubs(bundle)),
+        (Locations.LW_AFTER_MIDO_GRASS2, lambda bundle: can_cut_shrubs(bundle)),
+        (Locations.LW_AFTER_MIDO_GRASS3, lambda bundle: can_cut_shrubs(bundle)),
+        (Locations.LW_NEAR_SFM_GRASS1, lambda bundle: can_cut_shrubs(bundle)),
+        (Locations.LW_NEAR_SFM_GRASS2, lambda bundle: can_cut_shrubs(bundle)),
+        (Locations.LW_NEAR_SFM_GRASS3, lambda bundle: can_cut_shrubs(bundle)),
+    ])
+    # Connections
+    connect_regions(Regions.LW_BEYOND_MIDO, world, [
+        (Regions.LW_FOREST_EXIT, lambda bundle: True),
+        (Regions.LOST_WOODS, lambda bundle: is_child(bundle) or can_use(Items.SARIAS_SONG, bundle)),
+        (Regions.SFM_ENTRYWAY, lambda bundle: True),
+        (Regions.DEKU_THEATER, lambda bundle: True),
+        (Regions.LW_SCRUBS_GROTTO, lambda bundle: blast_or_smash(bundle)),
+    ])
+
+    # LW Near Shortcuts Grotto
+    # Locations
+    add_locations(Regions.LW_NEAR_SHORTCUTS_GROTTO, world, [
```

Comment:
This is missing the grotto events. Check out KF Storms Grotto inside kokiri_forest.py for the events you need. Just give them enums that fit this region instead of course.